contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/statsbeat.py

```diff
@@ -162,8 +185,12 @@ def _get_exception_count_value():
 
 class _StatsbeatMetrics:
 
-    def __init__(self, instrumentation_key):
-        self._instrumentation_key = instrumentation_key
+    def __init__(self, options):
+        self._options = options
+        self._instrumentation_key = options.instrumentation_key
+        self._feature = _StatsbeatFeature.NONE
+        if options.enable_local_storage:
+            self._feature |= _StatsbeatFeature.DISK_RETRY
```

Comment:
Is this something that can be turned off at some point?, if yes we need to update the value for following Statsbeat

Reply:
Cannot be turned off without restarting the SDK so feature flag cannot be changed during runtime.